gulpfile.js/tasks/server.js

```diff
@@ -1,11 +1,84 @@
 'use strict'
 
 const gulp = require('gulp')
+const portfinder = require('portfinder')
+const gutil = require('gulp-util')
 const browserSync = require('browser-sync')
-const config = require('../config').browserSync
+const colors = require('colors')
+const config = require('../config')
+const assetsPathV3 = `http://localhost:9032/public/v3-SNAPSHOT/`
+const assetsPathV4 = `http://localhost:9032/public/v4-SNAPSHOT/`
 
 gulp.task('server', () => {
-  config.forEach((conf, i) => {
-    browserSync.create(`${i}`).init(conf)
-  })
+  if (gutil.env.version === 'v3') {
+    browserSync.create().init(config.browserSync.assets, afStarted)
+    startServer('patternLibrary')
+      .then(() => {
+        startServer('componentLibrary')
```

Comment:
Not returning these promises is likely what's causing the output coming out before the end of the previous task.

We can look at improving this in another PR.